### DISCUSSION TITLE
Configurator on Pi 4

### DISCUSSION THREAD

**Discussion by Some-Won-1** [iNavFlight/inav#8239] [Q&A]
Can the Configurator run on a Pi 4 system?  I have not checked to see if the source can compile, but the Linux 64 version run time doesn't seem to work on Ubuntu Pi.

**Comment by stronnag** [comment]
> Can the Configurator run on a Pi 4 system? I have not checked to see if the source can compile, but the Linux 64 version run time doesn't seem to work on Ubuntu Pi.

No it cannot. The configurator is a x86_64 (amd64) application. It cannot run on aarch64 (or arm7/32). There is easy distribution for the required components for ARM (in particular or aarch64) and building an ARM target is a non-trivial task.

**Comment by Some-Won-1** [comment]
Thanks for the information.  I like a challenge.  Just what I thought having an amd64. I'll post if I get success.

**Comment by stronnag** [comment]
32bit is relatively easy as there is a NW.JS distribution with an SDK, aarch64 is more difficult, as there is no (obvious) NW.JS distribution with an SDK.

**Comment by Some-Won-1** [comment]
I attempted to launch it two different ways as was suggested in the Readme.MD write up. After installing Chromimum and a NW.js node, it would load into Chromimum extensions with two errors. The extension method gave a bluetooth error (...malformed something) and a svchost error.  Running the executable directly says it can not recognize the binary file. So now I'm learning how it works with Linux Ubuntu 22.04 before I come back to the Pi4 hardware.

The main reason I'm messing with the Pi is to get bluetooth to work and calibrate the gyro. My FC installation is too cramped to fit a USB cable inside. So far the Kakute H7 bluetooth is not cooperating with the Configurator using several flavors of OS. It will surprisingly connect with the Samsung S3 lite Android tablet using the SpeedyBee app, but I need the INAV Configurator.
